CI/windows/02_build_obs.ps1

```diff
@@ -64,8 +64,7 @@ function Configure-OBS {
     $GeneratorPlatform = "$(if (${BuildArch} -eq "x64") { "x64" } else { "Win32" })"
 
     $CmakeCommand = @(
-        "-S . -B `"${BuildDirectoryActual}`"",
-        "-G `"${CmakeGenerator}`"",
+        "-G", ${CmakeGenerator}
```

Comment:
Should this be quoted to handle generator names with spaces?

Reply:
Interestingly, no. It beaks when you wrap that Variable in quotes.

The issue is that Powershell applies its own "magic" quoting rules for command invocations. I won't pretend I entirely understood them, just know that it works when it's specified this way _exactly_ even though the string contains spaces.

Reply:
That's also why it's a separate token in the array, so Powershell considers the var (and its contents) in isolation.